lib/cgrates_web_jsonapi_web/views/cdr_stat_view.ex

```diff
@@ -13,4 +13,8 @@ defmodule CgratesWebJsonapiWeb.CdrStatView do
     :total_normal_clearing_disconnects,
     :total_rejected_disconnects
   ])
+
+  def render("extra_fields.json", %{data: data}) do
```

Comment:
I cannot understand why you have created a view and a render function inside cdr_stat_view. It looks like ExtraFieldsView is unused

I guess a better approach is creating a new controller CdrExtraFieldsController which will render a list of fields.